expression/aggregation/window_func.go

```diff
@@ -131,7 +131,8 @@ func (s *WindowFuncDesc) CanPushDownToTiFlash(ctx sessionctx.Context) bool {
 	}
 	// window functions
 	switch s.Name {
-	case ast.WindowFuncRowNumber, ast.WindowFuncRank, ast.WindowFuncDenseRank, ast.WindowFuncLead, ast.WindowFuncLag:
+	case ast.WindowFuncRowNumber, ast.WindowFuncRank, ast.WindowFuncDenseRank, ast.WindowFuncLead, ast.WindowFuncLag,
```

Comment:
I think we should check the frame type explicity here because `first_value/last_value` can use frame that is not supported by TiFlash yet.

Reply:
> I think we should check the frame type explicity here because `first_value/last_value` can use frame that is not supported by TiFlash yet.

I think we can ignore the check after this [pr](https://github.com/pingcap/tiflash/pull/7514#event-9366700483)

Reply:
> Even after that pr, TiFlash still not support range based frame.

okk, I will check frame type.

Reply:
> Even after that pr, TiFlash still not support range based frame.

As we can see core/exhaust_physical_plans.go:2767, the range frame type has been forbidden in the pushdown.